proto/p2p/conn_msgs.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package tendermint.proto.p2p;
+
+option go_package = "github.com/tendermint/tendermint/proto/p2p";
+
+import "third_party/proto/gogoproto/gogo.proto";
+
+message PacketPing {}
+
+message PacketPong {}
+
+message PacketMsg {
+  int32 channel_id = 1 [(gogoproto.customname) = "ChannelID"];
+  int32 eof        = 2 [(gogoproto.customname) = "EOF"];
+  bytes byte       = 3;
```

Comment:
```suggestion
  bytes data       = 3;
```

?

Reply:
ah was trying to keep parity with the go type, but data is more explanatory than` bytes` will rename the go type as well when the reactor is transitioned